### ISSUE TITLE
Does the validator include validation for ISO 19115-2:2019 including MI_Metadata? Or is this in the pipe-line?

### ISSUE THREAD

**Issue by MartinePaepen** [INSPIRE-MIF/helpdesk-validator#253]
Before reporting an issue with the INSPIRE validator deployed at JRC, please check the [list of known limitations](https://github.com/inspire-eu-validation/community/wiki/Known-limitations).

Please replace these lines with a brief summary of your issue and the expected
behaviour.

To help us investigate the issue, you may want to include:

- [ ] a description of the steps to reproduce your issue
- [ ] the browser and version used
- [ ] a screenshot illustrating the problem
- [ ] the Test Report (ideally as an attachment to the issue)
- [ ] the name of the Test that failed or better: change the `Level of Detail`
in the Report by clicking on `All Details`, then scroll down to the failed test
and copy the Link on the right-hand side of the `Assertion URI`
- [ ] the URL of your service / the file you have uploaded or referenced (as
an attachment).

If your service is protected or you have confidential data that you cannot
upload here, you can request an email address to which you can send URLs / files.
Please note that issues cannot be investigated for services/datasets that we cannot
access.

**Comment by MarcoMinghini** [comment]
Dear @MartinePaepen,
first, I am not sure which _update_ you mean in the title of the issue.

To also answer the same question asked in #65 and #130: TG 2.0 states that _new versions of the ISO 19115 standard were out of scope for this version of this specification_. There are currently no plans to change the TG, since ISO 19115:2003 is still widely used, and ISO 19115-2 is not.

However, some efforts should have already been made to establish a mapping from the old to the new MD standard. In case this is of interest, I suggest to start a discussion on the [INSPIRE Community Forum](https://inspire.ec.europa.eu/forum/).

**Comment by MartinePaepen** [comment]
Dear @MarcoMinghini ,
I was assuming that there was a new update. I see that this is not the case and changed the title.
We want to use the MI_Metadata schemes for including acquisitionInformation and that is why we already moved towards ISO 19115-2:2019. Now we want to perform the INSPIRE validation as we were used previously. And this doesn't work. We couldn't upload such a file and while reading #65, we decided to install the validator localy and include the gmi namespace. We are now able to upload the file however no validation is done AND this is not clear in the report. No error reported.
Kind regards,
Martine
